builtin/mainmenu/common.lua

```diff
@@ -77,7 +77,7 @@ function order_favorite_list(list)
 end
 
 --------------------------------------------------------------------------------
-function render_favorite(spec, is_favorite)
+function render_server(spec, is_favorite)
```

Comment:
oh, misread - I thought this function was the one that made the serverlist formspec in its entirety